manuscript/foreword.md

```diff
@@ -38,47 +38,52 @@ There are many [testimonials](https://roadtoreact.com/), [ratings](https://www.g
+**Is there any troubleshoot area?** 
+If you run into problems, please join the Slack Group. Also, check the [open issues on GitHub](https://github.com/rwieruch/the-road-to-learn-react/issues) to see if any solutions are listed for specific issue.  If your problem wasn't mentioned, open a new issue where you can explain your problem, provide a screenshot, and offer more details (e.g. book page, node version).
 
-**Is there a money back guarantee?** Yes, there is 100% money back guarantee for two months if you don't think it's a good fit. Please reach out to me to get a refund.
+**Can I help to improve the content?**
+Yes, I love to hear feedback. Simply open an issue on [GitHub](https://github.com/rwieruch/the-road-to-learn-react). These can be technical improvements, or clarification on the discussed topics. I am not a native speaker, so any feedback is appreciated. You can open pull requests on the GitHub page as well.
 
-**How to support the project?** If you believe in the content that I create, you can [support me](https://www.robinwieruch.de/about/). Furthermore, I would be grateful if you spread the word about this book after you read it and enjoyed reading it. Furthermore, I would love to have you as my [Patron on Patreon](https://www.patreon.com/rwieruch).
+**Is there a money back guarantee?**
+Yes, there is 100% money back guarantee for two months if you don't think it's a good fit. Please reach out to me to get a refund.
 
-**What's your motivation behind the book?** I want to teach about this topic in a consistent way. You often find material online that doesn't receive any updates or only teaches a small part of a topic. When you learn something new, people struggle to find consistent and up-to-date resources to learn from. I want to give you this consistent and up-to-date learning experience. In addition, I hope I can support minorities with my projects by giving them the content for free or by [having other impacts](https://www.robinwieruch.de/giving-back-by-learning-react/). In addition, in the recent time, I found myself fulfilled when teaching others about programming. It's a meaningful activity for me that I prefer over any other 9 to 5 job at any company. That's why I hope to pursue this path in the future.
+**How to support the project?**
+If you believe in the content I create, you can [support me](https://www.robinwieruch.de/about/). It also helps if you spread the word about this book, or you can sign on as my [Patron on Patreon](https://www.patreon.com/rwieruch).
 
-**Is there a call to action?** Yes. I want you to take a moment to think about a person who would be a good match to learn React. The person could have shown the interest already, could be in the middle of learning React or might not yet be aware about wanting to learn React. Reach out to that person and share the book. It would mean a lot to me. The book is intended to be given to others.
```

Comment:
I have added it back word for word.  I deleted this paragraph because I felt it wasn't truly a "frequently asked question"  No one really asks if there is a call to action about a programming book; the phrase "call to action" is generally reserved for causes like women's rights or raising money to fight a disease.  I also feel like you don't need to ask people who are already reading your book to share it with others.  It is high quality, so they will do that on their own.